### ISSUE TITLE
There should be a title on all html element to aid on hover tool tip and screen reader especially on action button

### ISSUE THREAD

**Issue by lykmapipo** [CodeTanzania/emis-dashboard#58]
![screenshot from 2018-09-25 14-07-01](https://user-images.githubusercontent.com/1610857/46010760-709d0480-c0cc-11e8-8f4c-caca3f53c63c.png)

Reference:
- [https://www.w3schools.com/tags/att_title.asp](https://www.w3schools.com/tags/att_title.asp)
